ToDoList/Views/ToDoView.swift

```diff
@@ -28,27 +27,33 @@ struct ToDoView: View {
                 Spacer()
                 Image(systemName: "plus")
```

Comment:
```suggestion
              Button {
                showAddTaskSheet = true
                        selectedTask = nil
            } label: {
                     Image(systemName: "plus")
                     .font(.title2)
            }
           
```

Instead of using a tap gesture on an image, you can use a button that has a label that's the image you want